src/ape_accounts/accounts.py

```diff
@@ -257,3 +259,39 @@ def __decrypt_keyfile(self, passphrase: str) -> HexBytes:
             return EthAccount.decrypt(self.keyfile, passphrase)
         except ValueError as err:
             raise InvalidPasswordError() from err
+
+
+def generate_account(
+    alias: str, passphrase: str, hd_path: str = ETHEREUM_DEFAULT_PATH, word_count: int = 12
+) -> Tuple[AccountAPI, str]:
+    """
+    Generate a new account.
+
+    Args:
+        extra_entropy (str): Extra entropy to use when generating the account.
+
+    Returns:
+        Tuple of AccountAPI and mnemonic for the generated account.
+    """
+    EthAccount.enable_unaudited_hdwallet_features()
+    accounts = AccountManager()
```

Comment:
I had to solve a similar issue here: https://github.com/ApeWorX/ape/pull/1888

So you can do:

```python
accounts = ManagerAccessMixin.account_manager
```

and import from the top:

```python
from ape.utils.basemodel import ManagerAccessMixin
```

At least, it fixed the issues I was seeing so I am assuming it should work here?